### PR TITLE
update README for npm packages

### PR DESCRIPTION
**Pull request by anurse**

### CODE REVIEW

File: clients/ts/signalr-protocol-msgpack/README.md [hunk 1]
```diff
@@ -15,9 +15,10 @@ To use the client in a browser, copy `*.js` files from the `dist/browser` folder
 ### Example (Browser)
 
 ```JavaScript
-let connection = new signalR.HubConnection('/chat', {
-    protocol: new signalR.protocols.msgpack.MessagePackHubProtocol()
-});
+let connection = new signalR.HubConnectionBuilder()
+    .withUrl("/chat")
+    .withHubProtocol(new signalR.protocols.msgpack.MessagePackHubProtocol())
```
Comment (davidfowl) [inline clients/ts/signalr-protocol-msgpack/README.md]: Double check this

Comment (anurse) [reply clients/ts/signalr-protocol-msgpack/README.md]: No, I just wrote it off the top of my head and didn't check at all

**Comment by anurse** [comment]
Quick update: added a link to the docs site (its out of date right now, but it's easier to update than the README)

File: clients/ts/signalr-protocol-msgpack/README.md (hunk 1 above)
Comment (anurse) [reply clients/ts/signalr-protocol-msgpack/README.md]: Ah, I see what you meant now. I had fixed this locally but not pushed it.

**Pull request merged**